.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+on:
+  - push
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - '3.8'
+          - '3.9'
+          - '3.10'
+          - '3.11'
+          - '3.12'
+          - '3.13'
+    steps:
+      - uses: actions/checkout@v4
+      - run: pip install numpy pytest
+      - run: pytest
+  docs:
+    if: false
```

Comment:
Skipping doc builds for now. Bringing this into the current decade is probably more important :) especially for such a tiny project.